### ISSUE TITLE
Extend

### ISSUE THREAD

**Issue by microdesign** [amsgames/laravel-shop#46]
I don't find a way to extend your 'placeOrder' method in my models or anywhere

**Comment by amsgames** [comment]
Is not a extend of any model. If you have configured the facade, you just
need to call it as a static function of class Shop

**Comment by microdesign** [comment]
I know this - I dont know how (I start work on one project where your package is used) and after this line
'$order = Shop::placeOrder();' code dont throw error but is not executed.

So I try to do something after this method create order, but code just break without error on this line.

**Comment by amostajo** [comment]
Is your cart filled?

**Comment by microdesign** [comment]
This is the peace of code 

`Shop::setGateway('customPaypalExpress');

```
    $cart = Cart::Current();
    $cart_total = $cart->total;

    $order = Shop::placeOrder();`
```

**Comment by amsgames** [comment]
CustomPaypalExpress, is that your custom gateway?

**Comment by microdesign** [comment]
Yes - I found something else interesting. I was able to add coupon code which is more than the cart order  and was able to try to pay which landed me to error.

**Comment by microdesign** [comment]
Yep this '$cart->addCoupon($coupon);' allow me to add coupon code which is more that total of order and I get one negative value lie '-$100'
